lighthouse-core/report/html/report-styles.css

```diff
@@ -478,47 +472,45 @@
 }
 
 
-.lh-metric .lh-metric__value::after {
+.lh-metric .lh-metric__innerwrap::before {
   content: '';
-  width: var(--lh-score-icon-width);
-  height: var(--lh-score-icon-width);
-  background-size: contain;
+  width: var(--shape-size);
+  height: var(--shape-size);
   display: inline-block;
-  vertical-align: text-bottom;
-  margin-left: calc(var(--body-font-size) / 2);
+  margin: var(--shape-margin);
 }
 
 .lh-metric--pass .lh-metric__value {
-  color: var(--pass-color);
+  color: var(--color-green-700);
 }
-.lh-metric--pass .lh-metric__value::after {
-  background: var(--pass-icon-url) no-repeat 50% 50%;
+.lh-metric--pass .lh-metric__innerwrap::before {
+  border-radius: 100%;
+  background: var(--color-green);
 }
 
-
 .lh-metric--average .lh-metric__value {
-  color: var(--average-color);
-  padding-left: 16px;
+  color: var(--color-orange-700);
 }
-.lh-metric--average .lh-metric__value::after {
-  background: var(--average-icon-url) no-repeat 50% 50%;
+.lh-metric--average .lh-metric__innerwrap::before {
+  background: var(--color-orange);
+  clip-path: var(--polygon-square);
```

Comment:
is this going to be a browser compat issue?

Reply:
havent tested in Edge, but apparently Edge support `clip-path` but only for svg urls. could transition to that.

Reply:
err.. but apparently safari has no support.

Reply:
yup safari can't do the triangle. Let's just fall back to normal css triangles? 
squares and circles are easy enough to do on their own.. i guess we miss out on using polygon() this time. :(

Reply:
done